torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -1087,7 +1087,7 @@ class TensorExprFuser {
 
     // A hook to optimizations limitter to allow bisecting the pass
     if (!JIT_OPT_ALLOWED) {
-      return false;
+      REQ(false && "jit opt limited");
```

Comment:
Does this belong to this PR?

Reply:
Yeah it's not strictly related but it was useful to me while bisecting so I thought I might as well leave it.

Reply:
Nit: probably could've rewritten it as `REQ(JIT_OPT_ALLOWED && "jit opt limited")`.

Reply:
Yeah good point that's cleaner.  I'll change to that!

Reply:
I'm actually not sure the macros magic is composable, but would be cool if it works :)